sklearn/grid_search.py

```diff
@@ -439,6 +440,10 @@ def predict(self, X):
             underlying estimator.
 
         """
+        check_is_fitted(self, 'best_estimator_',
+                        'This GridSearchCV instance was initialized with '
```

Comment:
Ah. Good catch. The error message should be different when not fitted and when `refit=False` I think. I'll push a fix...